homeassistant/components/system_bridge/sensor.py

```diff
@@ -283,15 +280,17 @@ async def async_setup_entry(
                 SystemBridgeSensor(coordinator, description, entry.data[CONF_PORT])
             )
 
-    displays = []
-    for display in coordinator.data.display.displays:
-        displays.append(
+    displays: list[dict] = []
```

Comment:
A dict is also a collection, so you should specify what is in it. For example, if it is a mapping of string with an integer:

```py
displays: list[dict[str, int]] = []
```